app/src/components/common/TmConnectedNetwork.vue

```diff
@@ -173,6 +181,14 @@ export default {
   background: var(--app-nav);
 }
 
+.noClickable {
+  cursor: unset;
+}
+
+.noClickable:hover {
```

Comment:
no no - if it's not clickable nothing should happen on hover

Reply:
yeah, that is why it is keeping the same color, overriding the previous hover. This is the idea